packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -76,33 +77,14 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       return null;
     }
 
-    // Loop over all the price feeds to medianize.
-    const priceFeeds = [];
-    for (const medianizedFeedConfig of config.medianizedFeeds) {
```

Comment:
This was refactored since we construct a MedianizerPriceFeed both here and for the `BasketSpreadPriceFeed`